site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -1111,3 +1179,11 @@ components:
                "id": "349556ea-4902-47c7-beb0-1009ab693fb4",
                "not-after": "2027-12-24T19:00:00Z",
             }]
+      FilterOptionsExample:
+         value:
+            enabled: true
+            ext.brand: "eclipse"
+      SortingOptionsExample:
+         value:
+            status.created: "ascending"
```

Comment:
I would expect the possible values for the sort direction to be defined somewhere, e.g. in an enum?